platform/android/detect.py

```diff
@@ -134,7 +134,7 @@ def mySpawn(sh, escape, cmd, args, env):
 		env['SHLINKFLAGS'][1] = '-shared'
 		env['SHLIBSUFFIX'] = '.so'
 	elif (os.name=="nt"):
-		gcc_path=gcc_path+"/windows-x86_64/bin" #this may be wrong
```

Comment:
Are you absolutely sure it's a "fix"? As far as I know there are two versions of NDK, 32bit and 64bit, 32bit has "windows" folder while "64" bit has "windows-x86_64". What you just did is made it look for 32 bit toolchain by default.